Rakefile

```diff
@@ -18,8 +18,16 @@
 #++
 
 require "yast/rake"
-# load the local tasks
-require_relative "lib/yast/rake_ci"
+
+begin
+  # load the local tasks
+  require_relative "lib/yast/rake_ci"
+rescue LoadError
+  # when building the RPM package at Jenkins the required gems might be missing
+  # at the create tarball step (they are installed later into the OSC chroot),
+  # do not fail in that case
+  $stderr.puts "Warning: Loading rake:ci task failed, skipping the task"
```

Comment:
maybe also put to stderr, that it is not problem on jenkins when building this package to not write false warning that can confuse someone